Chirp/Pages/ChatsPage/ChatsTableViewCell.swift

```diff
@@ -0,0 +1,128 @@
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    
+    // Setup tableview cell
+    func setup() {
+        setupViews()
+        setupConstraints()
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        friendImageView.layer.cornerRadius = friendImageView.frame.width/2
+        unreadMessageCounterLabel.layer.cornerRadius = unreadMessageCounterLabel.frame.width/2
+        
+    }
+    
+    // Add subviews
+    private func setupViews() {
+        addSubview(cellBoundaryview)
+        addSubview(friendImageView)
+        addSubview(fullNameLabel)
+        addSubview(lastMessageLabel)
+        addSubview(lastMessageTimeLabel)
+        addSubview(unreadMessageCounterLabel)
+    }
+    
+    // Configure constraints for the subviews
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            cellBoundaryview.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 24),
+            cellBoundaryview.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -24),
+            cellBoundaryview.topAnchor.constraint(equalTo: topAnchor),
+            cellBoundaryview.bottomAnchor.constraint(equalTo: bottomAnchor),
+            cellBoundaryview.heightAnchor.constraint(greaterThanOrEqualToConstant: imageViewWidth + 20),
+            
+            friendImageView.leadingAnchor.constraint(equalTo: cellBoundaryview.leadingAnchor),
+            friendImageView.centerYAnchor.constraint(equalTo: cellBoundaryview.centerYAnchor),
+            friendImageView.widthAnchor.constraint(equalToConstant: imageViewWidth),
+            friendImageView.heightAnchor.constraint(equalToConstant: imageViewWidth),
+            
+            fullNameLabel.leadingAnchor.constraint(equalTo: friendImageView.trailingAnchor, constant: leadingTrailingAnchor),
+            fullNameLabel.heightAnchor.constraint(equalToConstant: labelHeight),
+            fullNameLabel.topAnchor.constraint(equalTo: friendImageView.topAnchor, constant: leadingTrailingAnchor),
+            
+            lastMessageTimeLabel.trailingAnchor.constraint(equalTo: cellBoundaryview.trailingAnchor),
+            lastMessageTimeLabel.heightAnchor.constraint(equalToConstant: labelHeight),
+            lastMessageTimeLabel.topAnchor.constraint(equalTo: fullNameLabel.topAnchor),
+            
+            lastMessageLabel.leadingAnchor.constraint(equalTo: friendImageView.trailingAnchor, constant: leadingTrailingAnchor),
+            lastMessageLabel.heightAnchor.constraint(equalToConstant: labelHeight),
+            lastMessageLabel.bottomAnchor.constraint(equalTo: friendImageView.bottomAnchor, constant: -leadingTrailingAnchor),
+            
+            unreadMessageCounterLabel.trailingAnchor.constraint(equalTo: cellBoundaryview.trailingAnchor),
+            unreadMessageCounterLabel.heightAnchor.constraint(equalToConstant: labelHeight),
```

Comment:
Can you also add height constraint for the unreadMessageCounter just so that it is always a full circle?

Reply:
applied suggested changes